src/main/java/io/cdap/plugin/gcp/bigquery/sink/AbstractBigQuerySinkConfig.java

```diff
@@ -38,10 +38,15 @@
   @Nullable
   @Description("The Google Cloud Storage bucket to store temporary data in. "
     + "It will be automatically created if it does not exist, but will not be automatically deleted. "
-    + "Cloud Storage data will be deleted after it is loaded into BigQuery. " +
-    "If it is not provided, a unique bucket will be created and then deleted after the run finishes.")
+    + "Cloud Storage data will be deleted after it is loaded into BigQuery. "
+    + "If it is not provided, a unique bucket will be created and then deleted after the run finishes.")
   protected String bucket;
 
+  @Macro
+  @Description("Specifies the action that occurs if input field is missing in the destination table or "
```

Comment:
Whether to modify the BigQuery table schema if it differs from the input schema.

Reply:
Done.